src/main/java/org/spongepowered/api/text/title/Title.java

```diff
@@ -204,4 +207,390 @@ public String toString() {
                 .toString();
     }
 
+    /**
+     * Represents a builder class to create immutable {@link Title}
+     * configurations.
+     *
+     * @see Title
+     */
+    public static class Builder {
```

Comment:
I prefer having the static methods at the end of the file personally, does it really matter?